### DISCUSSION TITLE
Plans to add support for OAS 3.1.0

### DISCUSSION THREAD

**Discussion by guptavip20** [OpenAPITools/openapi-diff#380] [General]
I have been using Openapi-diff but recently team decided to move to OAS-3.1 but openapi-diff doesn't support 3.1.0 so is there any way you are planning to add support for 3.1.0.

**Comment by joschi** [comment]
See https://github.com/OpenAPITools/openapi-diff/discussions/345#discussioncomment-2280589:

> We're planning to support OpenAPI 3.1 as soon as the upstream projects swagger-core and swagger-parser support this version of the specification.
> 
> You can follow the development at [swagger-api/swagger-parser#1535](https://github.com/swagger-api/swagger-parser/issues/1535).

**Comment by DrSatyr** [comment]
just to heads up: https://github.com/swagger-api/swagger-parser/releases/tag/v2.1.0 release introduces support of OAS 3.1.0

**Comment by DrSatyr** [comment]
I've checked with basic PetStore 3.1 schemas. At least it works without any errors.
